stephanie-cirne-s05/projeto_casa.py

```diff
@@ -0,0 +1,195 @@
+                
+            if codigo_opcao not in [1, 2, 3, 4, 5, 6]:
+                print("Opção inválida. Por favor, escolha uma opção válida (1 a 5).\n")
+        except ValueError:
+            print("Entrada inválida. Por favor, digite um número inteiro.\n")
+            
+        return codigo_opcao
+
+
+#Inserindo tupla com nome e sobrenome que será a chave principal no dicionário nova aluna   
+def incluir_nova_aluna():
+    inserir_nome_aluna = str (input ("Insira o primeiro nome da aluna: "))
+    inserir_sobrenome_aluna = str (input ("Insira o sobrenome da aluna: "))
+    nova_aluna = (inserir_nome_aluna, inserir_sobrenome_aluna)
+    dataset [nova_aluna] = {}
+
+   
+
+    turma = incluir_turma (nova_aluna)
+    notas = lista_notas (nova_aluna)
+    presenca = lista_presenca(nova_aluna)
+    nota_particpacao = particpacao (nova_aluna)
+  
+
+    salva_dados (nova_aluna,turma, notas, presenca, nota_particpacao)
+
+
+    return nova_aluna
+
+
+def incluir_turma (nova_aluna):
+    print (f"A nova aluna {nova_aluna} foi incluída com sucesso \n")
+
+    turma = str (input ("Insira a turma da aluna: "))
+    dataset [(nova_aluna)] = [turma]
+
+    
+
+    print (f"A {turma} foi incluída no cadastro da aluna {nova_aluna}")
+
+    return turma
+
+def lista_notas (nova_aluna):
+
+    print ("Insira uma nota de 1 a 10 cada matéria: \n")
+    
+    lista_notas = []
+
+    for i in range (1,4):
+        valor = float (input (f"Insira uma nota de 1 a 10 para a matéria {i}: \n"))
+        lista_notas. append (valor)
+
+    dataset [(nova_aluna)] = [lista_notas]
+
+    print (f"As notas {lista_notas} foram incluídas no sistema! \n")
+
+    return lista_notas
+
+
+def lista_presenca (nova_aluna):
+
+    lista_presenca_2 =[] 
+
+    for i in range (1,6):
+        valor = input(f"A aluna esteve presnete na aula {i}? Insira 'True' para presença e 'False' para ausência em cada aula:  \n")
+        lista_presenca_2.append (valor)
+
+    dataset [(nova_aluna)] = [lista_presenca_2] 
+
+    print (f"A {lista_presenca_2} foi incluída no sistema com sucesso: ")
+
+    return lista_presenca_2
+
+
+def particpacao (nova_aluna):
+
+    nota_participacao = float (input ("Insira a nota de participação de 1 a 10: "))
+
+    dataset [(nova_aluna)]  = [nota_participacao]
+
+    print (f"A nota de participação {nota_participacao} foi incluída com sucesso: ")
+
+    return nota_participacao
+
+def salva_dados (nova_aluna,incluir_turma, lista_notas,lista_presenca, participacao):
+    
+    dataset [nova_aluna] = {
+        "Turma": incluir_turma,
+        "Notas": lista_notas,
+        "Presença": lista_presenca,
+        "Participação": participacao
+
+    }
+    print ("Dados da aluna salvos com sucesso!")
+
+#-------------------------------------------------------------------------------------------------------  
+   
+  
+def consultar_lista_alunas():
+    for nome,sobrenome in (dataset.keys()):
+        print ((nome,sobrenome))
```

Comment:
Como você desempacotou a tupla para ser usado aqui, podemos usar o nome e sobrenome sem precisar usar parênteses:

**Sugestão:**
```suggestion
    for nome, sobrenome in dataset.keys():
        print(f'{nome} {sobrenome}')
```